### ISSUE TITLE
Check if test does not have assert(s).

### ISSUE THREAD

**Issue by JohanLarsson** [nunit/nunit.analyzers#112]
For catching mistakes like:

```cs
[Test]
public void SomeTest()
{
    // Assert.AreEqual(...);
}
```

Perhaps very common.

Don't warn if `[Ignore]` or `[Explicit]`

**Comment by Dreamescaper** [comment]
Not sure is possible.
Asserts might be present somewhere in user code, not nessesary directly in test method.
Besides, external assertion libs (e.g. FluentAssertions) would be a pain as well, I suppose

**Comment by jnm2** [comment]
I use Shouldly a lot these days, so this would be every test method for me.
Maybe an opt-in diagnostic would make sense?

**Comment by JohanLarsson** [comment]
The usefulness is perhaps negative, not a common bug. We had a guy who had a habit for commenting out  the body of test methods or asserts that were troublesome 🙂

**Comment by jnm2** [comment]
Empty test method bodies sound like a good thing to warn for. What do you think?

**Comment by JohanLarsson** [comment]
Yes it is a trivial analyzer with nonzero value.
